zigzag/zigzag.py

```diff
@@ -13,201 +13,202 @@
 from swagger_client.rest import ApiException
 
 
-# ======================================================================================================================
-# Globals
-# ======================================================================================================================
-TESTCASE_NAME_RGX = re.compile(r'(\w+)(\[.+\])')
-TESTCASE_GROUP_RGX = re.compile(r'tests\.(test_[\w-]+)\.?(Test\w+)?$')
-MAX_FILE_SIZE = 52428800
-
-
-# ======================================================================================================================
-# Functions
-# ======================================================================================================================
-def _load_input_file(file_path, pprint_on_fail=False):
-    """Read and validate the input file contents.
-
-    Args:
-        file_path (str): A string representing a valid file path.
-        pprint_on_fail (bool): A flag for enabling debug pretty print on schema failure.
-
-    Returns:
-        ElementTree: An ET object already pointed at the root "testsuite" element.
-
-    Raises:
-        RuntimeError: invalid path.
-    """
-
-    root_element = 'testsuite'
-    junit_xsd = pytest_rpc.get_xsd()
-
-    try:
-        if os.path.getsize(file_path) > MAX_FILE_SIZE:
-            raise RuntimeError("Input file '{}' is larger than allowed max file size!".format(file_path))
-
-        junit_xml_doc = etree.parse(file_path)
-    except (IOError, OSError):
-        raise RuntimeError("Invalid path '{}' for JUnitXML results file!".format(file_path))
-    except etree.ParseError:
-        raise RuntimeError("The file '{}' does not contain valid XML!".format(file_path))
-
-    try:
-        xmlschema = etree.XMLSchema(etree.parse(junit_xsd))
-        xmlschema.assertValid(junit_xml_doc)
-        junit_xml = junit_xml_doc.getroot()
-    except etree.DocumentInvalid as e:
-        debug = "\n\n---DEBUG XML PRETTY PRINT---\n\n"
-        error_message = "The file '{}' does not conform to schema!" \
-                        "\n\nSchema Violation:\n{}".format(file_path, str(e))
-        if pprint_on_fail:
-            error_message = "{0}{1}{2}{1}".format(error_message,
-                                                  debug,
-                                                  etree.tostring(junit_xml_doc, pretty_print=True))
-        raise RuntimeError("The file '{}' does not conform to schema!"
-                           "\n\nSchema Violation:\n{}".format(file_path, error_message))
+class ZigZag:
+
+    _testcase_name_rgx = re.compile(r'(\w+)(\[.+\])')
+    _testcase_group_rgx = re.compile(r'tests\.(test_[\w-]+)\.?(Test\w+)?$')
+    _max_file_size = 52428800
```

Comment:
Since these are all constants that should not be changed during runtime the names should be in all caps. `_MAX_FILE_SIZE`